docs/hacking_howto.md

```diff
@@ -108,7 +108,7 @@ First, make sure you have run the "Create some data" step above.
 We include some sample data to get you started. You can install it by
 running this command::
 
-    $ ./manage.py generatedata
+    docker-compose exec web ./manage.py generatedata
```

Comment:
I would suggest to move the whole section `Install Sample Data` above the `Get Search working`. The latter leaves the user into a bash shell into the container and running `docker-compose ...` in there won't work. By moving it up we can avoid this scenario

Reply:
But in my humble opinion, `Install Sample Data` subsection is more suitable under `Further Setup` section. Not everyone needs to generate sample data, it's an additional step for those who wants to work on Kitsune instance which contains some data. What's your thought now? If still, wants stick to your proposed changes, I'm more than happy to make those changes.

Reply:
Content wise I agree with what you are saying. All I am saying is that the user might get confused by this command b/c the last step left the user inside the container. Another solution here could be to add the `docker-compose exec web` command to the search part too for consistency.

Reply:
Actually, `docker-compose exec web` command already mentioned in [Get search working](https://kitsune.readthedocs.io/en/latest/hacking_howto.html#get-search-working) section, see first instruction. I'm thinking about another solution, adding one another step (4th step) to exit from `web`'s bash shell. Something like,

4. Now exit from web's bash shell: `exit`

and adding `docker-compose exec web` command to other remaining sections, also like [Product Details Initialization](https://kitsune.readthedocs.io/en/latest/hacking_howto.html#product-details-initialization), [Pre-compiling JavaScript Templates](https://kitsune.readthedocs.io/en/latest/hacking_howto.html#pre-compiling-javascript-templates). What do you think? Is it a good idea?

Reply:
sounds good!